src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,94 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String createReportFull = readFromFile(fromFileName);
+        String createReport = createReport(createReportFull);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(createReportFull);
```
Уделяй больше внимания названиям переменных. 
То что ты читаешь из файда - это просто информация. Там нет никакого репорта.
Метод называется createReport потому что он создает Report, это правильно. Но зачем в репорте, который он уже создал слово create?